client/ayon_traypublisher/plugins/publish/collect_clip_instances.py

```diff
@@ -9,14 +9,30 @@ class CollectClipInstance(pyblish.api.InstancePlugin):
     order = pyblish.api.CollectorOrder - 0.081
 
     hosts = ["traypublisher"]
-    families = ["plate", "review", "audio"]
+    families = [
```

Comment:
NOTE: Maybe `CollectEditorialInstance` should add one family (e.g. `"editorial.clip"`) to it's children instances which would be used here for filtering, instead of having `families` and `creator_identifier` filtering...

Reply:
CollectEditorialInstances is adding editorial product with OTIO or EDL representations. That is not a place where I could influence clip products.